lib/imgur.js

```diff
@@ -337,7 +339,20 @@ imgur.deleteImage = async (deleteHash) => {
     throw new Error('Missing delete hash');
   }
 
-  return await imgur._imgurRequest('delete', 'deleteHash');
+  return await imgur._imgurRequest('delete', deleteHash);
```

Comment:
Had we had tests for this method, we would have caught this obvious error. Now, we do have tests!

Reply:
wow! tsk tsk @ myself 😞